components/raftstore/src/store/peer_storage.rs

```diff
@@ -663,7 +663,8 @@ where
 
 #[derive(Debug, PartialEq)]
 pub enum RaftlogFetchState {
-    Fetching,
+    // The Instant records the start time of the fetching.
+    Fetching(Instant),
```

Comment:
I think you need to remove `PartialEq` derive as it's not a valid implementation anymore.

Reply:
Fixed.